src/genericComponents/Checkbox.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import { InputGroup } from 'react-bootstrap';
+import styled from 'styled-components';
+
+const Checkbox = (props: any) => {
+  const [clicked, setClicked] = useState(false);
```

Comment:
This state doesn't use our props or communicate outside of the checkbox, so it can't help us update state in a form, for example. You can remove this altogether